core/windows_and_tabs/window_snap.py

```diff
@@ -275,11 +287,13 @@ class Actions:
     def snap_window(pos: RelativeScreenPos) -> None:
         """Move the active window to a specific position on-screen.
 
-        See `RelativeScreenPos` for the structure of this position.
-
-        """
+        See `RelativeScreenPos` for the structure of this position."""
```

Comment:
I would just delete this line entirely, the type information makes this redundant